docs/en/samples/data_co_locality.md

```diff
@@ -99,6 +99,10 @@ spec:
     alluxio.user.local.reader.chunk.size.bytes: 256MB
     alluxio.worker.network.reader.buffer.size: 256MB
     alluxio.user.streaming.data.timeout: 300sec
+  fuse:
+    args:
+      - fuse
+      - --fuse-opts=direct_io,ro,max_read=131072,attr_timeout=7200,entry_timeout=7200,nonempty
```

Comment:
Can we use `--fuse-opts=kernel_cache,ro,max_read=131072,max_readahead=0,attr_timeout=7200,entry_timeout=7200,nonempty` 
 here?

Reply:
Done